tasks/build/package_scripts/pre_install.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+if command -v systemctl >/dev/null && systemctl is-active kibana.service >/dev/null; then
+    systemctl --no-reload stop kibana.service
+fi
+if [ -x /etc/init.d/kibana ]; then
+    if command -v invoke-rc.d >/dev/null; then
+        invoke-rc.d kibana stop
+    elif command -v service >/dev/null; then
```

Comment:
I see. In that case why do we need to check whether the kibana service is running on line 3 with systemctl? Does line 4 end up invoking a different stop function than the one in kibana/build/services/sysv/init.d/kibana?